CONST_Makefile

```diff
@@ -305,7 +307,6 @@ GPG_KEYS += 4E98BB3AD36285EFDA47B83B4662C60FA510717D # Michael Künzli
 GPG_KEYS += 1AC7B924F73B4C4A302F39E6EA57F57AA68C2F5F # Arnaud Morvan
 GPG_KEYS += 0545262E5AE04436C83B654D4CEC60DAC4F68132 # Andrea Borghi
 GPG_KEYS += B3771480D024F905F6FA4637BE875E27AAA3DDF0 # Renata Mueller
-GPG_KEYS += C1599283494601AA15A502C9002B1BA74D3386D2 # Benjamin Gerber
```

Comment:
You removed your GPG key ?

Reply:
It was an error, I've added a last commit.